test/starknet/ControllerActions.test.ts

```diff
@@ -149,7 +149,7 @@ describe('Controller Actions', () => {
       user.address,
       utils.intsSequence.IntsSequence.LEFromString(''),
       vanillaExecutionStrategy.address,
-      [vanillaVotingStrategy.address],
+      ['0x2'],
```

Comment:
looking at those `0x0`, `0x1`, `0x2`, it might be a good idea to declare a variable `latestIndex` and increment it and use it when need be? Might make working with this code easier in the future

Reply:
feels like a lot of effort right now lol, will defo do that for the starknet.js scripts